tests/compiled_templates/bird.cfg

```diff
@@ -0,0 +1,99 @@
+# Generated by confd
```

Comment:
I can, I did note that in my initial PR comment that this will add the overhead of maintaining both the templates as well as the data.